### DISCUSSION TITLE
Radio field data not being included in edit submission view

### DISCUSSION THREAD

**Discussion by rellafella** [solspace/craft-freeform#1316] [Questions & Answers] [answered]
### What happened?

We have a radio field and are not using the "Use Custom values" option so the values just end up coming through the same as the label.

I can see that there is data in the database for the field in the submission table, but when we look at the submission in the CP there is nothing selected.

I presume it is something to do with special characters being included.

an example of one of the labels is
30 – 70 cm (~cattle dog nose tip to tail)

### How can we reproduce this?

1. Create a radio field
2. Add options with special characters and spaces etc
3. Don't select "use custom options"
4. Submit the form, selecting one of the options
5. Check submissions in cp
6. Confirm data in db table

### Freeform Edition

Pro

### Freeform Version

4.1.14

### Craft Version

Craft Pro 4.9.5

### When did this issue start?

Unsure

**Comment by kjmartens** [comment]
Hi @rellafella,

I'm sorry about the trouble you're experiencing. I can duplicate this issue as well when I use strings with special characters like the example you gave (`30 – 70 cm (~cattle dog nose tip to tail)`). However, is there any reason why you cannot create a safer value such as `30_70cm` or `30_70cm_cattle-dog-nose-tip-to-tail` for these option values? I think it will help in the long run to have a simpler value for the option value and then the descriptive label for the option label. 🙂 

If you're concerned about accessing the descriptive option label (instead of value) inside an email notification template or Submissions query, you can use `submission.[field.handle].label` or `submission.firstName.label`, etc. 🙂

**Reply by rellafella** [reply]
The problem that I am faced with is that clients are often creating their own forms and are not aware of which characters are "safe". Not sure what the solution would be here, but I don't think it is expecting non-techical people using a form builder to validate their own inputs.

**Reply by kjmartens** [reply]
Thank you for your feedback. I forgot to mention that I have noted this for our developer team to see if there's a quick fix we can make here. 🙂

**Reply by rellafella** [reply]
Thanks @kjmartens!

**Comment by seandelaney** [comment]
Fixed with PR's pending 

https://github.com/solspace/craft-freeform/pull/1321 (v5)
https://github.com/solspace/craft-freeform/pull/1322 (v4)

**Comment by kjmartens** [comment, marked as answer]
The issue seemed to be specifically with _en_ and _em_ dashes. This is now resolved in Freeform 4.1.19 and 5.3.2. 🙂
